plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -42,14 +53,52 @@ public void execute(@CanManage User deputy, Guild guild, String[] args) {
         when(args.length < 1, this.messages.generalNoNickGiven);
         when(guild.getMembers().size() >= this.config.maxMembersInGuild, formatter.format(this.messages.inviteAmount));
 
-        User invitedUser = UserValidation.requireUserByName(args[0]);
+        if (args[0].equals(this.config.inviteCommandAllArgument)) {
+
+            Double range;
+
+            try {
+                range = Math.pow(Double.parseDouble(args[1]), 2);
+            }
+            catch (Exception exception) {
+                range = this.config.inviteCommandAllMaxRange;
+            }
+
+            Double finalRange = range;
+            List<Player> nearbyPlayers = Bukkit.getServer().getOnlinePlayers().stream()
+                    .filter(player -> finalRange >= player.getLocation().distanceSquared(sender.getLocation()))
+                    .filter(player -> player != sender)
+                    .collect(Collectors.toList());
+
+            when(guild.getMembers().size() + nearbyPlayers.size() > this.config.maxMembersInGuild, formatter.format(this.messages.inviteAmount));
+            when(nearbyPlayers.size() == 0, this.messages.inviteNoOneIsNearby);
+
+
+            for (Player player : nearbyPlayers) {
+                User invitedUser = this.userManager.findByPlayer(player).get();
+
+                if (invitedUser.isVanished()) {
+                    continue;
+                }
+
+                this.inviteUserToGuild(deputy, guild, invitedUser, formatter);
+            }
+
+            return;
+
+        }
+
+        this.inviteUserToGuild(deputy, guild, UserValidation.requireUserByName(args[0]), formatter);
+    }
+
+    private void inviteUserToGuild(User deputy, Guild guild, User invitedUser, FunnyFormatter formatter) {
 
         if (this.guildInvitationList.hasInvitation(guild, invitedUser)) {
             if (!SimpleEventHandler.handle(new GuildMemberRevokeInviteEvent(EventCause.USER, deputy, guild, invitedUser))) {
                 return;
             }
 
-            deputy.sendMessage(this.messages.inviteCancelled);
+            deputy.sendMessage(this.messages.inviteCancelled.replace("{USER}", invitedUser.getName()));
```

Comment:
Użyj FunnyFormatter'a

Reply:
Formatter fajny ale nie do tego PR.

Reply:
To tak trochę słaby plan - to jest jedna zmienna, więc replace może być. Chyba, że jest jakaś zasada, że w FG nie używamy String#replace

Reply:
Tak, używamy tutaj FunnyFormatter'a żeby mieć replace w 1 miejscu